PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+- [ ] Tested via `nix.useChroot`.
+- [ ] Compiled on all platforms written in `meta` section.
+- [ ] Compiled on other platform(s): .
+- [ ] Tested compilation of all pkgs that depend on this change.
+- [ ] Was executed in runtime.
```

Comment:
`Tested package` IMHO doesn't make it very clear it's supposed to be runtime testing.